docs/app/Examples/views/Statistic/Types/StatisticExampleGroupColored.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Statistic } from 'semantic-ui-react'
+
+const items = [
+  { label: 'Faves', value: '22' },
+  { label: 'Views', value: '31,200' },
+  { label: 'Members', value: '22' },
+]
+
+const StatisticExampleGroupColored = () => (
+  <div>
+    <Statistic.Group items={items} color='blue' />
+  </div>
```

Comment:
Since there is only one element here, let's forgo the extra `div` wrapper and just return the `<Statistic.Group />` itself.